### ISSUE TITLE
Lint job does not run against proposed PR changes

### ISSUE THREAD

**Issue by stmcginnis** [vmware-tanzu/tanzu-framework#1793]
**Bug description**

It was noticed in https://github.com/vmware-tanzu/tanzu-framework/pull/1512 that there were changes to the `lint` target, but those changes were not reflected in the job execution. It looks like this is because the Lint job does not perform the checkout action, so it doesn't actually run against the proposed changes.

**Affected product area (please put an X in all that apply)**

- [ ] APIs
- [ ] Addons
- [ ] CLI
- [ ] Docs
- [ ] IAM
- [ ] Installation
- [ ] Plugin
- [ ] Security
- [X] Test and Release
- [ ] User Experience

**Comment by vuil** [comment]
@stmcginnis thanks for filing the issue, looks like from recently closed PRs you might be still looking into this?

**Comment by stmcginnis** [comment]
Yeah, thanks for checking. What I thought was the cause (and fix) ended up not being the case. So it might be good to keep this open so if others run into something similar they can report that here and we can collect more data.

It is possible this was related to the issue being addressed in https://github.com/vmware-tanzu/tanzu-framework/pull/1786
